src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl

```diff
@@ -256,7 +259,7 @@
 
 
   <!-- Distribution format: Show list of allowed formats -->
-  <xsl:template mode="mode-iso19139" match="//gmd:distributionInfo/gmd:MD_Distribution/gmd:distributionFormat/gmd:MD_Format/gmd:name" priority="2005">
+  <xsl:template mode="mode-iso19139" match="//gmd:distributionInfo/gmd:MD_Distribution/gmd:distributionFormat/gmd:MD_Format/gmd:name[$schema = 'iso19139.ca.HNAP']" priority="2005">
```

Comment:
To check if can be better to remove this template and use the `data-gn-keyword-picker` in `config-editor.xml`, like:

```
  <for name="gmd:name" xpath="//gmd:distributionInfo/gmd:MD_Distribution/gmd:distributionFormat/gmd:MD_Format/gmd:name"
         use="data-gn-keyword-picker">
      <directiveAttributes data-thesaurus-key="external.theme.EC_Resource_Formats" data-focus-to-input="false"/>

  </for>
```

Reply:
I will not action this in this PR -- lets move this out later...